go/build.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
```

Comment:
What do you think about putting all this in the Makefile? Just want to avoid having different scripts that pretty much act the same way :smile:

Reply:
In general I think that Makefiles are a pool tool. For this particular case it wouldn't make much difference because the tasks are very simple. But things always get complicated over time, and that usually results in Makefiles that are hard to understand and maintain. I'd rather gradually replace the existing Makefile with a better tool. But if you still prefer the Makefile I will use it. Just let me know.

Reply:
Let's solve the complicated case once we get there. For now we have the `Makefile` (for better or for worse). Let's use it :pray: